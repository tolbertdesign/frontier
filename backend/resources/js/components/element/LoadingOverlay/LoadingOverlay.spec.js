import { shallowMount, createLocalVue } from '@vue/test-utils'
import LoadingOverlay from './LoadingOverlay.vue'
import Buefy from 'buefy'

const localVue = createLocalVue()
localVue.use(Buefy, {
  defaultIconPack: 'fas',
})

const isLoading = true
const isFullPage = true

describe('LoadingOverlay', () => {
  it('Snapshop test for LoadingOverlay', () => {
    const wrapper = shallowMount(LoadingOverlay, {
      localVue,
      propsData: {
        isLoading: isLoading,
        isFullPage: isFullPage,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
