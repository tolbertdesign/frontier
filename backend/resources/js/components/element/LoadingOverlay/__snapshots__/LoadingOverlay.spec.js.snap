// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingOverlay Snapshop test for LoadingOverlay 1`] = `<bloading-stub active="true" isfullpage="true" animation="fade" cancancel="true" oncancel="function _default() {}"></bloading-stub>`;
